src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1250,6 +1250,9 @@ public function getFormat($mimeType)
             if (in_array($mimeType, (array) $mimeTypes)) {
                 return $format;
             }
+            if (isset($canonicalMimeType) && in_array($canonicalMimeType, (array) $mimeTypes)) {
```

Comment:
I would probably rather initialise `$canonicalMimeType` with `null` and omit the `isset()`.